City-of-Helsinki/drupal-module-helfi-proxy
# UHF-3861: Redirect to proxy domain

To test this:

- Setup local proxy for your project: https://github.com/City-of-Helsinki/drupal-helfi-local-proxy
- Run `composer require drupal/helfi_proxy:dev-UHF-3861`
- Add `$config['helfi_proxy.settings']['default_proxy_domain'] = 'helfi-proxy.docker.so';` to your `public/sites/default/local.settings.php`

Make sure you get redirected to `helfi-proxy.docker.so` when opening any page using the default domain (like `helfi-kymp.docker.so`)
